svgpath.pegjs

```diff
@@ -20,6 +20,20 @@
   // The letter of the last parsed command.
   var lastCh = '';
 
+  // Flatten an array.
+  function flatten(a) {
+    var flat = [];
+    for (var i = 0; i < a.length; i++) {
+      if (a[i] instanceof Array) {
+        flat.push.apply(flat, flatten(a[i]));
+      }
```

Comment:
Let's make this:

  } else {

to keep the formatting consistent with the rest of the file.

Besides this one nit, this looks great to me, thanks!